systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -478,10 +478,12 @@ protected void testDockerImagesForKafkaCluster(String clusterName, String cluste
             kafkaVersion = Environment.ST_KAFKA_VERSION;
         }
 
-        //Verifying docker image for zookeeper pods
-        for (int i = 0; i < zkPods; i++) {
-            String imgFromPod = PodUtils.getContainerImageNameFromPod(kafkaNamespaceName, KafkaResources.zookeeperPodName(clusterName, i), "zookeeper");
-            assertThat("Zookeeper pod " + i + " uses wrong image", imgFromPod, containsString(TestUtils.parseImageMap(imgFromDeplConf.get(KAFKA_IMAGE_MAP)).get(kafkaVersion)));
+        if (!isKRaftEnabled) {
```

Comment:
I guess you can use the `Environment.isKRaftModeEnabled()` directly here and you don't have to pass it as a method parameter.

(Same applies for the second if)